### PR TITLE
Add persistent_notification automation trigger to docs

### PR DESCRIPTION
**Pull request by RoboMagus**
## Proposed change

Updated docs for `persistent_notification` automation triggers.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#94809
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/automation/templating.markdown [hunk 1]
```diff
@@ -144,6 +144,20 @@ These are the properties available for a [Time Pattern trigger](/docs/automation
...
 | `trigger.now` | DateTime object that triggered the time_pattern trigger.
 
+### Persistent Notification
+
+These are the properties available for a [Persistent Notification trigger](/docs/automation/trigger/#persistent-notification-trigger).
+
+| Template variable | Data |
+| ---- | ---- |
+| `trigger.platform` | Hardcoded: `persistent_notification`
+| `trigger.update_type` | Type of persistent notification update `added` or `removed`.
```
Comment (bdraco) [inline source/_docs/automation/templating.markdown]: This needs to be updated

File: source/_docs/automation/templating.markdown [hunk 2]
```diff
@@ -144,6 +144,20 @@ These are the properties available for a [Time Pattern trigger](/docs/automation
...
+| `trigger.notification` | Notification object that triggered the persistent_notification trigger.
+| `trigger.notification.notification_id` | The notification ID
+| `trigger.notification.title` | Title of the notification
+| `trigger.notification.message` | Message of the notification
+| `trigger.notification.created_ad` | DateTime object indicating when the notification was created.
```
Comment (bdraco) [inline source/_docs/automation/templating.markdown]: typo?

Comment (bdraco) [inline source/_docs/automation/templating.markdown]:
```suggestion
| `trigger.notification.created_at` | DateTime object indicating when the notification was created.
```

File: source/_integrations/persistent_notification.markdown [hunk 3]
```diff
@@ -18,6 +18,33 @@ The `persistent_notification` integration can be used to show a notification on
   <img src='/images/screenshots/persistent-notification.png' />
 </p>
 
+## Automation
+
+Persistent Notification [Triggers](/docs/automation/trigger) enable automations to be
+triggered when `persistent_notification`s are `added` or `removed`.
```
Comment (bdraco) [inline source/_integrations/persistent_notification.markdown]: This looks like it needs to be updated as well

Comment (RoboMagus) [reply source/_integrations/persistent_notification.markdown]: Expanded the wording for all options for the `persistent_notification` trigger.

File: source/_docs/automation/templating.markdown [hunk 4]
```diff
@@ -144,6 +144,20 @@ These are the properties available for a [Time Pattern trigger](/docs/automation
 | `trigger.platform` | Hardcoded: `time_pattern`
 | `trigger.now` | DateTime object that triggered the time_pattern trigger.
 
+### Persistent Notification
+
+These are the properties available for a [Persistent Notification trigger](/docs/automation/trigger/#persistent-notification-trigger).
```
Comment (bdraco) [inline source/_docs/automation/templating.markdown]:
```suggestion
These properties are available for a [Persistent Notification trigger](/docs/automation/trigger/#persistent-notification-trigger).
```

File: source/_docs/automation/templating.markdown [hunk 5]
```diff
@@ -144,6 +144,20 @@ These are the properties available for a [Time Pattern trigger](/docs/automation
...
 
+### Persistent Notification
+
+These properties are available for a [Persistent Notification trigger](/docs/automation/trigger/#persistent-notification-trigger).
+
+| Template variable | Data |
+| ---- | ---- |
+| `trigger.platform` | Hardcoded: `persistent_notification`
+| `trigger.update_type` | Type of persistent notification update `added`, `removed`, `current` or `updated`.
+| `trigger.notification` | Notification object that triggered the persistent_notification trigger.
```
Comment (bdraco) [inline source/_docs/automation/templating.markdown]:
```suggestion
| `trigger.notification` | Notification object that triggered the persistent notification trigger.
```

File: source/_integrations/persistent_notification.markdown [hunk 6]
```diff
@@ -18,6 +18,33 @@ The `persistent_notification` integration can be used to show a notification on
   <img src='/images/screenshots/persistent-notification.png' />
 </p>
 
+## Automation
+
+Persistent Notification [Triggers](/docs/automation/trigger) enable automations to be
+triggered when `persistent_notification`s are updated. This can be limited to a specific notification by providing an ID for `notification_id`, or when this value is omitted the automation will trigger for any notification ID. If no `update_type` is provided, the automation will trigger for any of the following update types: `added`, `removed`, `updated`, `current`. By providing one or more of these values to the `update_type` option, the automation triggers only on these events.
+Review the [Automating Home Assistant](/getting-started/automation/)
+getting started guide on automations or the [Automation](/docs/automation/)
+documentation for full details.
```
Comment (bdraco) [inline source/_integrations/persistent_notification.markdown]:
```suggestion
Persistent Notification [Triggers](/docs/automation/trigger) enable automations to be triggered when persistent notifications are updated. Triggers can be limited to a specific notification by providing an ID for `notification_id`, or when this value is omitted the automation will trigger for any notification ID. If no `update_type` is provided, the automation will trigger for the following update types: `added`, `removed`, `updated`, `current`. By providing one or more of these values to the `update_type` option, the automation triggers only on these `update_type` events.

Review the [Automating Home Assistant](/getting-started/automation/) getting started guide on automations or the [Automation](/docs/automation/) documentation for full details.
```

**Review by bdraco — DISMISSED** [review]

File: source/_docs/automation/templating.markdown [hunk 7]
```diff
@@ -144,6 +144,20 @@ These are the properties available for a [Time Pattern trigger](/docs/automation
...
 | `trigger.now` | DateTime object that triggered the time_pattern trigger.
 
+### Persistent Notification
+
+These properties are available for a [Persistent Notification trigger](/docs/automation/trigger/#persistent-notification-trigger).
+
+| Template variable | Data |
+| ---- | ---- |
+| `trigger.platform` | Hardcoded: `persistent_notification`
+| `trigger.update_type` | Type of persistent notification update `added`, `removed`, `current` or `updated`.
```
Comment (frenck) [inline source/_docs/automation/templating.markdown]:
```suggestion
| `trigger.update_type` | Type of persistent notification update `added`, `removed`, `current`, or `updated`.
```

File: source/_docs/automation/trigger.markdown [hunk 8]
```diff
@@ -777,6 +778,22 @@ Do not prefix numbers with a zero - using `'01'` instead of `'1'` for example wi
 
 </div>
 
+## Persistent Notification trigger
```
Comment (frenck) [inline source/_docs/automation/trigger.markdown]:
```suggestion
## Persistent notification trigger
```

File: source/_docs/automation/trigger.markdown [hunk 9]
```diff
@@ -19,6 +19,7 @@ An automation can be triggered by an event, a certain entity state, at a given t
 - [Template trigger](#template-trigger)
 - [Time trigger](#time-trigger)
 - [Time pattern trigger](#time-pattern-trigger)
+- [Persistent Notification trigger](#persistent-notification-trigger)
```
Comment (frenck) [inline source/_docs/automation/trigger.markdown]:
```suggestion
- [Persistent notification trigger](#persistent-notification-trigger)
```

File: source/_docs/automation/trigger.markdown [hunk 10]
```diff
@@ -777,6 +778,22 @@ Do not prefix numbers with a zero - using `'01'` instead of `'1'` for example wi
...
 
+## Persistent Notification trigger
+
+Persistent notification triggers are fired when a `persistent_notification` is `added` or `removed` that matches the configuration options.
+
+```yaml
+automation:
+  trigger:
+    - platform: persistent_notification
+      update_type:
```
Comment (frenck) [inline source/_docs/automation/trigger.markdown]:
```suggestion
      update_type:
```

File: source/_integrations/persistent_notification.markdown [hunk 11]
```diff
@@ -18,6 +18,31 @@ The `persistent_notification` integration can be used to show a notification on
   <img src='/images/screenshots/persistent-notification.png' />
 </p>
 
+## Automation
+
+Persistent Notification [Triggers](/docs/automation/trigger) enable automations to be triggered when persistent notifications are updated. Triggers can be limited to a specific notification by providing an ID for `notification_id`, or when this value is omitted the automation will trigger for any notification ID. If no `update_type` is provided, the automation will trigger for the following update types: `added`, `removed`, `updated`, `current`. By providing one or more of these values to the `update_type` option, the automation triggers only on these `update_type` events.
```
Comment (frenck) [inline source/_integrations/persistent_notification.markdown]:
```suggestion
Persistent notification [triggers](/docs/automation/trigger) enable automations to be triggered when persistent notifications are updated. Triggers can be limited to a specific notification by providing an ID for `notification_id`, or when this value is omitted the automation will trigger for any notification ID. If no `update_type` is provided, the automation will trigger for the following update types: `added`, `removed`, `updated`, or `current`. By providing one or more of these values to the `update_type` option, the automation triggers only on these `update_type` events.
```

File: source/_integrations/persistent_notification.markdown [hunk 12]
```diff
@@ -18,6 +18,31 @@ The `persistent_notification` integration can be used to show a notification on
...
+
+Review the [Automating Home Assistant](/getting-started/automation/) getting started guide on automations or the [Automation](/docs/automation/) documentation for full details.
+
+{% my automations badge %}
+
+An example of a persistent_notification trigger in YAML:
```
Comment (frenck) [inline source/_integrations/persistent_notification.markdown]:
```suggestion
An example of a persistent notification trigger in YAML:
```

**Review by frenck — APPROVED** [review]

**Pull request merged**